avocado/utils/vmimage.py

```diff
@@ -32,6 +32,13 @@
 from . import process
 
 
+#: The "qemu-img" binary used when creating the snapshot images.  If
+#: set to None (the default), it will attempt to find a suitable binary
+#: with :func:`avocado.utils.path.find_command`, which uses the the
+#: system's PATH environment variable
```

Comment:
@clebergnu, Out of curiosity: Why `#:` on comments?

Reply:
The variable gets automatically documented with that comment style.